docs/source/docs/explanation/formats/imagenet.md

```diff
@@ -39,38 +39,41 @@ Load ImageNet dataset through the Python API:
 import datumaro as dm
 
 dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagenet_txt')
+# or
+dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagent')
```

Comment:
```suggestion
dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagenet')
```

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/985/commits/8e5ebb817a14b8e73fac486a1171bbac5e8dc740